### PR TITLE
Resolve algolia build failures

### PR DESCRIPTION
**Pull request by msh100**

### CODE REVIEW

**Comment by Tonnu** [comment]
lgtm 👍

File: docs-sidebar/index.js [hunk 1]
```diff
@@ -60,7 +59,7 @@ function renderSidebar (props, state, setState) {
 
   var input;
 
-  var base = getWindowUrl();
+  const base = getWindowUrl();
```
Comment (flenter) [inline docs-sidebar/index.js]: Looks like support is fairly ok for const (http://caniuse.com/#search=const)
However, there are some issues with safari though and there's no babel/transpiling happening for docs afaik. I would prefer to use var here still.

Comment (msh100) [inline docs-sidebar/index.js]: @flenter Interesting, I am also in favour of var as I don't see the advantage here. It's a const to be inline with the rest of the document.

Comment (flenter) [inline docs-sidebar/index.js]: Right, I only looked at the code for the pr. Nevermind then!

**Pull request merged**